_projects/fluffychat_voip.md

```diff
@@ -0,0 +1,46 @@
+---
+name: FluffyChat webrtc
+desc: Make webrtc support in FluffyChat ready for daily use
+requirements:
+ - Familiar with Dart/Flutter
+difficulty: high
+issues:
+ - https://gitlab.com/famedly/fluffychat/-/issues/874
+mentors:
+ - Krille
+initiatives:
+ - GSoC
+tags:
```

Comment:
Can you add a `175h`or `350h` tag depending on whether you think it will be a short or large project :) ?

Reply:
Done :)